ui/mainframe.py

```diff
@@ -154,23 +154,23 @@ def __init__(self, mainUI=None):
         self.takeBackupBtn.configure(highlightcolor="black")
         self.takeBackupBtn.configure(pady="0")
         self.takeBackupBtn.configure(text='''Take Backup''')
-
-        userVar = tk.StringVar(mainUI, "User")
-        adminVar = tk.StringVar(mainUI, "Admin")
+        self.userVar = tk.StringVar(mainUI, "User")
+        self.adminVar = tk.StringVar(mainUI, "Admin")
```

Comment:
rename adminVar to some meaningful name becasuse we are using this for user and admin both.
We can use `userMode` as variable name

Reply:
ok, how about radio_admin

Reply:
or about, user_type

Reply:
great